### PR TITLE
Update meter creation

### PR DESCRIPTION
**Pull request by gannan08**

### CODE REVIEW

File: lib/http.js [hunk 1]
```diff
@@ -335,10 +337,24 @@ async function _getMeterClient() {
 async function _createMeter({controller, productId}) {
   const {url} = config['profile-http'].meterService;
 
+  // create root capability
+  const capability = `urn:zcap:root:${encodeURIComponent(url)}`;
+
+  // create signer using the application's capability invocation key
+  const {keys: {capabilityInvocationKey}} = getAppIdentity();
+  const invocationSigner = capabilityInvocationKey.signer();
```
Comment (dlongley) [inline lib/http.js:345]: Did you consider using ezcap instead? I think that's the preferred method now.

Comment (gannan08) [reply lib/http.js]: Sure: https://github.com/digitalbazaar/bedrock-profile-http/pull/39/commits/dbd4b6377db7470404d0ead9acd1cdc32bbde45f

File: lib/http.js [hunk 2]
```diff
@@ -7,13 +7,28 @@ const {asyncHandler} = require('bedrock-express');
...
-const {httpClient} = require('@digitalbazaar/http-client');
+const {getAppIdentity} = require('bedrock-app-identity');
 const {httpsAgent} = require('bedrock-https-agent');
 const {profiles, profileAgents} = require('bedrock-profile');
 const {validate} = require('bedrock-validation');
+const {ZcapClient} = require('@digitalbazaar/ezcap');
+
+let ZCAP_CLIENT;
+
+bedrock.events.on('bedrock.init', async () => {
```
Comment (dlongley) [inline lib/http.js:22]:
```suggestion
bedrock.events.on('bedrock.init', () => {
```

**Review by dlongley — APPROVED** [review]
Thanks!

**Pull request merged**